### PR TITLE
Support json format output and id query for get-default-channels

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/Darc/Options/GetDefaultChannelsCommandLineOptions.cs [hunk 1]
```diff
@@ -18,4 +19,13 @@ internal class GetDefaultChannelsCommandLineOptions : CommandLineOptions<GetDefa
     [Option("channel", HelpText = "Filter by a channel name. Matches on substring.")]
     public string Channel { get; set; }
 
+    [Option("ids", Separator = ',', HelpText = "Get only default channels with these ids.")]
```
Comment (premun) [inline src/Microsoft.DotNet.Darc/Darc/Options/GetDefaultChannelsCommandLineOptions.cs:22]:
```suggestion
    [Option("ids", Separator = ',', HelpText = "Get only default channels with these comma-separated IDs.")]
```

File: src/Microsoft.DotNet.Darc/Darc/Operations/GetDefaultChannelsOperation.cs [hunk 2]
```diff
@@ -77,4 +78,17 @@ public override async Task<int> ExecuteAsync()
             return Constants.ErrorCode;
         }
     }
+
+    private async Task<IEnumerable<DefaultChannel>> FilterDefaultChannels() => (await _barClient.GetDefaultChannelsAsync())
+                    .Where(defaultChannel =>
+                    {
+                        return  (_options.Ids == null || !_options.Ids.Any() || _options.Ids.Any(id => id.Equals(defaultChannel.Id.ToString()))) &&
```
Comment (premun) [inline src/Microsoft.DotNet.Darc/Darc/Operations/GetDefaultChannelsOperation.cs:85]: Should we compare the guids case insensitive too?

**Review by premun — DISMISSED** [review]

File: src/Microsoft.DotNet.Darc/Darc/Operations/GetDefaultChannelsOperation.cs (hunk 2 above)
Comment (mmitche) [reply src/Microsoft.DotNet.Darc/Darc/Operations/GetDefaultChannelsOperation.cs:85]: I wonder why GUIDs were chosen for subscriptions, but not for anything else.

Comment (mmitche) [reply src/Microsoft.DotNet.Darc/Darc/Operations/GetDefaultChannelsOperation.cs:85]: Maybe something to do with the actor state keys

**Review by premun — APPROVED** [review]

**Pull request merged**